data-prepper-plugins/parse-json-processor/build.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id 'java'
+}
+
+group 'org.opensearch.dataprepper'
+version '2.0.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
+
+    implementation project(':data-prepper-api')
+    implementation project(':data-prepper-plugins:common')
+    implementation 'com.fasterxml.jackson.core:jackson-databind'
+}
+
+test {
```

Comment:
Thanks for the recommendation! 

I set the threshold to 96% but it's slightly higher in reality because there aren't any unit tests for the `prepareForShutdown`, `isReadyForShutdown`, and `shutdown` methods. These methods are needed to extend `AbstractProcessor` but they're empty because `parse_json` doesn't have any special behavior for shutdown. The implemented line and branch coverage are 100%.